src/net_processing.cpp

```diff
@@ -515,8 +515,10 @@ struct Peer {
     /** Set of txids to reconsider once their parent transactions have been accepted **/
     std::set<uint256> m_orphan_work_set GUARDED_BY(g_cs_orphans);
 
+    /** Protects vRecvGetData **/
```

Comment:
ba951812ec0cc8ebee5911a582f188525b76ff0a

To describe a member `//!` is [recommended](https://github.com/bitcoin/bitcoin/blob/master/doc/developer-notes.md#coding-style-doxygen-compatible-comments):
```suggestion
    //! Protects vRecvGetData.
```